articles/libraries/lock-android/v1/native-social-authentication.md

```diff
@@ -0,0 +1,78 @@
+---
+toc_title: Native Social Authentication
+description: How to implement native social authentication with Lock Android
+---
+# Lock Android: Native Social Authentication
+
+<%= include('../_includes/_lock-version') %>
+
+**Lock** by default handles all social authentication with a Browser installed in your Android device, but for some social connections you can take advantage of our native integration.
```

Comment:
I don't know if I would add this article as this feature is no longer working. You can also keep it but add a warning like the one on this file https://auth0.com/docs/libraries/lock-android/native-provider-facebook